atomix/cluster/src/main/java/io/atomix/raft/partition/impl/RaftNamespaces.java

```diff
@@ -60,6 +62,8 @@
           .register(ConfigureResponse.class)
           .register(ReconfigureRequest.class)
           .register(ReconfigureResponse.class)
+          .register(TransferRequest.class)
```

Comment:
This breaks compatibility. You have to add new classes at the end so that the ID for existing classes are not changed.

Reply:
@deepthidevaki Good point I fixed it